.gatling/package.conf

```diff
@@ -0,0 +1,2 @@
+# See `example.package.conf` for a full deployment configuration file example
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```

Comment:
No space before semi colon:

```suggestion
# Learn more about Configuration as Code: https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```

Reply:
Still here

Reply:
I'm playing with the new PR comments integration in IDEA, it seems I failed 😄